VisualizationBase/src/Scene.cpp

```diff
@@ -109,11 +112,24 @@ ModelRenderer* Scene::defaultRenderer()
 	return &defaultRenderer_;
 }
 
-void Scene::addTopLevelItem(Item* item)
+void Scene::addTopLevelNode(Node *node, int column, int row)
+{
+	currentViewItem()->insertNode(node, column, row);
+}
+
+void Scene::addTopLevelItem(Item* item, bool show)
 {
 	Q_ASSERT(!inAnUpdate_);
+	//If we have a ViewItem, we must add it to all ViewItems
```

Comment:
I.E. Views should not be added via addTopLevelItem to the viewItems_ but rather adding to the viewItems should call addTopLevelItem which should work as before.